docs/observability/aws/integrations/aws-rds.md

```diff
@@ -0,0 +1,395 @@
+| parse regex field=query_block "# User@Host:\s*\S+?\[(?<user>\S*?)\]\s*@\s*(?<host_name>\S+)\s\[(?<ip_addr>\S*?)\]\s+Id:\s*(?<Id>\d+)"
+| where user != "rdsadmin" and !isEmpty(user) and user matches "*"
+| where !isEmpty(ip_addr) and ip_addr matches "*"
+| parse regex field=query_block "# Query_time:\s+(?<query_time>[\d.]*)\s+Lock_time:\s+(?<lock_time>[\d.]*)\s+Rows_sent:\s+(?<rows_sent>[\d]*)\s+Rows_examined:\s+(?<rows_examined>[\d]*)" nodrop
+| parse regex field=query_block "SET timestamp=(?<set_timestamp>\d*);\n(?<sql_cmd>[\s\S]*);" nodrop
+| parse regex field=sql_cmd "[^a-zA-Z]*(?<sql_cmd_type>[a-zA-Z]+)\s*"
+| avg(query_time) as avgTime, sum(query_time) as totalTime, min(query_time) as minTime, max(query_time) as maxTime, avg(rows_examined) as avgRowsExamined, avg(rows_sent) as avgRowsSent, avg(Lock_Time) as avgLockTime, count as frequency group by sql_cmd, user, ip_addr
+| sort by avgTime | limit 100
+```
+```sql title="Audit Logs (CloudWatch log based)"
+account=* region=* dbidentifier=* namespace=aws/rds _sourceHost=/aws/rds*Audit CONNECT
+| json "message" nodrop | if (_raw matches "{*", message, _raw) as message
+| parse field=message ",*,*,*,*,*,*,*,*,*" as instance, user, host, f1, f2, action, database, f3, f4 nodrop
+| where user matches "*" and host matches "*"
+| where action = "CONNECT"
+| count as eventCount
+```
+
+```sql title="General Logs (CloudWatch log based)"
+account=* region=* dbidentifier=* namespace=aws/rds _sourceHost=/aws/rds*general Connect
+| json "message" nodrop | if (_raw matches "{*", message, _raw) as message
+| parse regex field=message "\s*\d+\s+(?<cmdType>\S+)\s*(?<command>.*)"
+| where cmdType = "Connect"
+| parse field=command "*@* on  using *" as user, host, connectionType nodrop
+| parse field=command "*@* on * using *" as user, host, database, connectionType nodrop
+| parse field=command "Out\t*@*:*" as user, host, port nodrop
+| parse field=message "Access denied for user '*'@'*' (using *: *)" as user, host, autenticationType, flag nodrop
+| if (message matches "*Access denied*", "Fail", "Success") as connectionStatus
+| count as count by user
+| sort by count, user asc | limit 20
+```
+
+## Viewing the RDS dashboards  
+
+We highly recommend you view these dashboards in the [Explore View](/docs/observability/aws/deploy-use-aws-observability/view-dashboards) of the AWS Observability solution.
+
+### 1. Amazon RDS Overview
+
+The **Amazon RDS Overview** dashboard provides insights into RDS resource statistics and utilization throughout your infrastructure, including CPU, memory, latency, storage, and network throughput.
+
+Use this dashboard to:
+* Get a high-level overview of your Amazon RDS infrastructure.
+* Quickly identify problems in resource utilization.
+* Monitor databases performance insights such as relative CPU load, non-CPU load, and overall database load.
+
+<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-Overview.png')} alt="Amazon RDS dashboard" />
```

Comment:
```suggestion
<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-Overview.png')} alt="Amazon RDS dashboard" style={{border: '1px solid black'}} />
```

Can you add border to these images? Use `style={{border: '1px solid black'}}` inside the image tag.